### DISCUSSION TITLE
msgstore-YYYY-MM-DD msgstore-increment

### DISCUSSION THREAD

**Discussion by alefello** [KnugiHK/WhatsApp-Chat-Exporter#83] [Q&A]
Hello everybody
I'm stupid, so I saved the local backup folder of whatsapp (the one in sd card) instead of the msgstore.db file in /data folder. I've the key too. Is it possible to open all the backup chunks at once, to view all the chats together?
Thank you

**Comment by KnugiHK** [comment]
Hi. You just need to rename the latest file to msgstore.db.crypt(whatever it was).

**Comment by alefello** [comment]
Yeah, I noticed that the "android-backup" version merges all the backup
chunks (I suspect from the result). I'm sorry for the stupid question I
asked, but from the description and from the command I thought it would
have exported only the backup file msgstore.db.crypt14 and not the other
chunks. Despite that the resulting folder has too many chats and they're
too old to be only the first chunk exported I think.
Thank you so much

Il giorno lun 12 feb 2024 alle ore 03:52 Knugi ***@***.***>
ha scritto:

**Reply by KnugiHK** [reply]
AFAIK, the msgstore.db.crypt14 should be the latest backup and include every single message at the time when the backup was created. In other words, other files are the old backup.
